src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -344,7 +355,17 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
                 )}
               </Box>
             </Join>
-          </ModalDialog>
+            <Box>
+              <Button
+                disabled={!hasSelectedContactMethod || !isPhoneValid}
+                loading={isSubmitting}
+                onClick={() => handleSubmit()}
+                width="100%"
+              >
+                Save Alert
+              </Button>
+            </Box>
+          </Flex>
```

Comment:
WIP: we need to make this button sticky on the bottom of the Modal. Suggestions welcome (ideally we could keep it in the modal component?)

Reply:
Why can't we keep the implementation you added? ^^^

Reply:
Ah I see now